contracts/ProtocolReserve/RiskFundV2.sol

```diff
@@ -146,7 +146,8 @@ contract RiskFundV2 is AccessControlledV8, RiskFundV2Storage, IRiskFund {
 
     /// @notice Function to sweep token from pool
     /// @param tokenAddress Address of the asset(token)
-    /// @param comptroller Pool address to which assets will be transferred
+    /// @param comptroller Pool address that the assets belong to
+    /// @param receiver The receiver of the funds
     /// @param amount Amount need to sweep from the pool
     /// @custom:event Emits sweepTokenFromPool event on success
     /// @custom:error ZeroAddressNotAllowed is thrown when tokenAddress/comptroller address is zero
```

Comment:
I would add `@custom:access Controlled by AccessControlManager`, and I would open a PR in the docs repo to update this page, with the new natspec comments: https://docs-v4.venus.io/technical-reference/reference-isolated-pools/risk-fund-and-shortfall/risk-fund-v2